Automattic/wp-calypso
# Hide long data format via tooltip

Related to SELFSERVE-613

## Proposed Changes

* Make the date format shorted, and
* Hide the long version under a tooltip.

## Testing Instructions



* Run Calypso locally and opens Tools > Advertising.
* Ensure date format looks the same as in [Figma](https://www.figma.com/file/uvh2KtY65b7hGBHI9exbJ1/BlazePress?type=design&node-id=8572-76417&t=vSRZCWOBuy4WLzwg-0).

![date-tooltip](https://github.com/Automattic/wp-calypso/assets/115007291/b4749e82-78dc-4d02-a665-42b58ac216b6)

* (Optional) To limit unit tests to run only the new tests you can adjust [`package.json:116`](https://github.com/Automattic/wp-calypso/blob/trunk/package.json#L116) to become:
```json
"test-client": "TZ=UTC jest -c=test/client/jest.config.js -i client/my-sites/promote-post-i2/test/utils.test.tsx",
```

* Run unit tests to make sure the new test is working fine by running (if there's a way to run specified unit tests without changing `package.json` please let me know):
```sh
yarn run test-client
``` 

## Pre-merge Checklist
